app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<main>
+<nav class="sidebar">
+<h1>
+  TaskList
+</h1>
+<div class="clearfix"></div>
```

Comment:
This "navigation" section would make sense to have as part of the layout.  

Also maybe instead of `div` use `nav`.